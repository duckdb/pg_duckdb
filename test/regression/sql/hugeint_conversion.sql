create table hugeint_sum(a int);
insert into hugeint_sum select g from generate_series(1,100) g;
select pg_typeof(sum(a)) from hugeint_sum;
select sum(a) result from hugeint_sum;

drop table hugeint_sum;
