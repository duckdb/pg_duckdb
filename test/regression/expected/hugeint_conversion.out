create table hugeint_sum(a int);
insert into hugeint_sum select g from generate_series(1,100) g;
select pg_typeof(sum(a)) from hugeint_sum;
 pg_typeof 
-----------
 hugeint
(1 row)

select sum(a) result from hugeint_sum;
 result 
--------
   5050
(1 row)

drop table hugeint_sum;
